optimade/filtertransformers/mongo.py

```diff
@@ -138,14 +157,16 @@ def set_op_rhs(self, arg):
             return {"$in": arg[1:]}
 
         if arg[1] == "ALL":
-            raise NotImplementedError
+            return {"$all": arg[2]}
+
         if arg[1] == "ANY":
-            raise NotImplementedError
+            return {"$in": arg[2]}
+
         if arg[1] == "ONLY":
-            raise NotImplementedError
+            return {"$all": arg[2], "$size": len(arg[2])}
 
         # value with OPERATOR
-        raise NotImplementedError
+        raise NotImplementedError(f"set_op_rhs not implemented for {arg}.")
```

Comment:
Maybe a more informant line here, like
```suggestion
        raise NotImplementedError(f"set_op_rhs not implemented for use with OPERATOR. Given: {arg}")
```